examples/advanced_matplotlib.ipynb

```diff
@@ -0,0 +1,213 @@
+      "text/plain": [
+       "<Figure size 720x288 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, axs = plt.subplots(1, 2, figsize=(10, 4))\n",
+    "# force a diverging color map to use the full color space, not just upper half\n",
+    "# (as if it is not diverging)\n",
+    "traj_contacts.residue_contacts.plot_axes(ax=axs[0], cmap='PRGn', diverging_cmap=False);\n",
+    "\n",
+    "# force sequential color map to use only the upper half (as if it is diverging)\n",
+    "traj_contacts.residue_contacts.plot_axes(ax=axs[1], cmap='Blues', diverging_cmap=True);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b6b6acd1",
+   "metadata": {},
+   "source": [
+    "### \"Clipping\" at high or low values\n",
+    "\n",
+    "You might be interested in somehow marking which values are very high or very low. This can be done by creating a custom color map. Details on this can be found [in the matplotlib documentation on colormap manipulation](https://matplotlib.org/stable/tutorials/colors/colormap-manipulation.html). The basic idea we'll implement here is to use the `'PRGn'` colormap, but to make values below about -0.9 show up as red, and values above 0.9 show up as blue. This is very similar in principle to one of the matplotlib examples on [\"Creating listed colormaps\"](https://matplotlib.org/stable/tutorials/colors/colormap-manipulation.html#creating-listed-colormaps)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "dacabaa9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from matplotlib import cm\n",
+    "from matplotlib.colors import ListedColormap\n",
+    "import numpy as np\n",
+    "\n",
+    "PRGn = cm.get_cmap('PRGn', 180)\n",
+    "red = np.array([1.0, 0.0, 0.0, 1.0])\n",
+    "blue = np.array([0.0, 0.0, 1.0, 1.0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "9d223a64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a custom color map with bottom 5% red; top 5% blue; rest is normal PRGb\n",
```

Comment:
this confused me a bit the first time I read the percentages. maybe go for something like
```suggestion
    "# create a custom color map with bottom 0.1 red; top 0.1 blue; rest is normal PRGb\n",
```

Reply:
I made a slightly different change (emphasizing 200 colors, 10 red, 10 blue, rest PrGn). More significantly, I expanded the discussion in the Markdown text before this code to make it much clearer what's actually going on here. Please have another look.